app/__init__.py

```diff
@@ -19,6 +21,15 @@
 app.wsgi_app = ReverseProxy(app.wsgi_app, script_name='/')
 
 
+# Add quick log of the routes used to all request.
+# Important: this should be the first before_request method, to ensure
+# a failure in another pre request method would stop logging.
+@app.before_request
+def log_route():
+    g.setdefault('request started', time.time())
```

Comment:
Oops not sure if it is also the case on the template but here the `_` is missing; `request_started`

Reply:
nope, it's only in my code. I'll correct this